docs/source/reference/index.rst

```diff
@@ -1,6 +1,55 @@
 API Reference
 =============
 
+
+How to read Optuna API documentation?
+-------------------------------------
+
+We provide the API reference that is automatically generated by
+docstrings written in each modules. We basically follow `Google Style Python Docstrings <https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html>`_.
+For more information, please read our `Coding Style Conventions <https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#docstrings>`_ as well.
+A typical example of our docstring looks like following:
+
+.. code-block:: python
+
+    def hoge(self, number: int, ...) -> int:
```

Comment:
A minor suggestion. `foo` or perhaps something slightly more verbose might be better. `hoge` is quite domestic to Japan.
```suggestion
    def some_function(self, number: int, ...) -> int:
```

Reply:
I didn't know that..., thank you for letting me know! :memo: And adding `_function` also improve the readability. 🙏